src/explorer/trivy_result.ts

```diff
@@ -53,8 +53,8 @@ const processResult = (result: any): TrivyResult[] => {
 			let endLine = 1;
 
 			if (element.IacMetadata !== null) {
```

Comment:
I think this `if` needs removing as it will be `null` now?

Reply:
fixed - good spot